app/views/widgets/show.html.erb

```diff
@@ -0,0 +1,54 @@
+<head>
+  <%= stylesheet_link_tag "widget" %>
+</head>
+<body>
+  <div class="alaveteli-widget">
+    <div class="alaveteli-widget__top">
+      <div class="alaveteli-widget__left">
+        <div class="alaveteli-widget__title">
+          <%= link_to @info_request.title, request_path(@info_request), :target => "_top" %>
+        </div>
+        <div class="alaveteli-widget__status <%= @info_request.calculate_status %>">
+          <p class="alaveteli-widget__status__status-label">Status</p>
+          <%= status_description(@info_request) %>
```

Comment:
We're calling `@info_request.calculate_status` twice (L11, L13).

If we used an [`InfoRequestStatusPresenter`](https://github.com/mysociety/alaveteli/pull/1957/files#r28501316) we could cache the result of this.